package.json

```diff
@@ -100,7 +100,7 @@
   },
   "scripts": {
     "test-ci": "gulp ci-tests",
-    "test": "rimraf ./build && tsc && mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 60000 ./build/compiled/test",
+    "test": "ts-node test/utils/test-runner.ts",
```

Comment:
I would like to keep previous setup. `ts-node` works very slow in windows (in my own tests) and I don't want to use it.

Reply:
it was few years ago last time I reproduced issue on windows however. Maybe we can keep ts-node

Reply:
Well, let me know. We could support both if desired.